data/templates/install-ubuntu/post-install-ubuntu.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # create SSH key for root
-<% if (undefined !== rootSshKey) { -%>
+<% if ('undefined' !== typeof rootSshKey && null !== rootSshKey) { -%>
```

Comment:
You can shorten this if you want to either:

`hasOwnProperty("rootSshKey")` or `'undefined' != typeof rootSshKey` (!= non-strict comparison matches both undefined and null).

But this is also more explicit so it works as well :)